src/Tools/RapidResponseDesk/sections/AreaTime/AreaAndTimeSection.jsx

```diff
@@ -1,19 +1,234 @@
+      ];
+    });
+  };
+
+  const updateTimespan = (id) => (fromTime, toTime) => {
+    setTimespanArray((prevState) => {
+      return prevState.map((item, index) => (index === id ? { ...item, from: fromTime, to: toTime } : item));
+    });
+  };
+
+  const closeAllCalendarDialogs = () => {
+    setTimespanArray((prevState) => {
+      return prevState.map((item, index) => ({
+        ...item,
+        displayCalendarFrom: false,
+        displayCalendarTo: false,
+      }));
+    });
+  };
+
+  const updateCalendarOpenState = (selectedTimespan, state, isFrom = true) => {
+    if (state) {
+      closeAllCalendarDialogs();
+    }
+
+    setTimespanArray((prevState) => {
+      return prevState.map((item, index) =>
+        index === selectedTimespan.id
+          ? {
+              ...item,
+              displayCalendarFrom: isFrom ? state : false,
+              displayCalendarTo: !isFrom ? state : false,
+            }
+          : item,
+      );
+    });
+  };
+
+  const removeTimespanFromList = (id) => {
+    setTimespanArray((prevState) => prevState.filter((item, index) => index !== id));
+  };
+
+  const setTimespanPicker = (timespan, index) => (
+    <TimespanPicker
+      id="aoi-time-select"
+      minDate={minDateRange}
+      maxDate={maxDateRange}
+      datePickerInputStyle={{ width: '85px' }}
+      timespan={{ fromTime: timespan.from, toTime: timespan.to }}
+      applyTimespan={updateTimespan(index)}
+      timespanExpanded={true}
+      calendarHolder={cardHolderRef}
+      timespanLimit={(selectedTimespan) => detectIfDateIsOverlapping(index, selectedTimespan)}
+      displayCalendarFrom={timespan.displayCalendarFrom}
+      openCalendarFrom={() => updateCalendarOpenState(timespan, true)}
+      closeCalendarFrom={() => updateCalendarOpenState(timespan, false)}
+      displayCalendarUntil={timespan.displayCalendarTo}
+      openCalendarUntil={() => updateCalendarOpenState(timespan, true, false)}
+      closeCalendarUntil={() => updateCalendarOpenState(timespan, false, false)}
+      showNextPrevDateArrows={true}
+      getAndSetNextPrevDateFrom={async (direction, selectedDay) =>
+        await getAndSetNextPrevDate(direction, selectedDay, index, timespan)
+      }
+      getAndSetNextPrevDateTo={async (direction, selectedDay) =>
+        await getAndSetNextPrevDate(direction, selectedDay, index, timespan, false)
+      }
+      isDisabled={false}
+    />
+  );
+
+  const setDateRangeContainer = () => (
+    <div className="date-picker-container">
+      {timespanArray.map((timespanFrame, index) => {
+        return (
+          <div
+            key={`date-picker-${index}`}
+            className={`date-picker-content${index + 1 < timespanArray.length ? ' bottom-border' : ''}`}
+          >
+            {setTimespanPicker(timespanFrame, index)}
+            <div className="remove-button-container">
+              <Button
+                style={{ boxShadow: 'none', color: 'black' }}
+                icon={'fas fa-trash'}
```

Comment:
> Feels like the trash icon is not completely centered in the middle of the circle when you hover over it

Let me know if this is still an issue. I snipped it out and use Kruler and did not detect any specific irregularities.

Reply:
If you remove this small 2px gap at the bottom, it will be perfectly aligned I believe

![image](https://github.com/user-attachments/assets/22b3f14d-d1e1-4d5c-92e6-14e5083c0388)

Reply:
Hmm. That is interesting. On my browsers does not look like this. 
![image](https://github.com/user-attachments/assets/99732412-c0ba-4f34-90be-839942728d2b)
![image](https://github.com/user-attachments/assets/e92a7526-02d3-4ba5-914a-5431764899fe)

Reply:
Let me know if this last commit has fixed the problem.

Reply:
Yes, this looks great now

Reply:
Got info from the designer, the buttons now sadly look like they are disabled, so a new change is requested😅 make the icons blue, so they look like all the other buttons